core/src/test/java/org/infinispan/xsite/AbstractTwoSitesTest.java

```diff
@@ -107,6 +111,11 @@ protected void adaptLONConfiguration(BackupConfigurationBuilder builder) {
       //no-op
    }
 
+   public AbstractTwoSitesTest use2Pc(boolean use2Pc) {
+      this.use2Pc = use2Pc;
+      return this;
+   }
```

Comment:
This should be grouped with the other parameters (and with `parameters()` as well).